surefire-booter/src/main/java/org/apache/maven/surefire/booter/ForkedBooter.java

```diff
@@ -38,11 +38,16 @@
 import org.apache.maven.surefire.suite.RunResult;
 import org.apache.maven.surefire.testset.TestSetFailedException;
 
+import static java.lang.System.*;
```

Comment:
I really don't like this static import nor the next one: System.out and System.err in code is a lot more manageable that out and err with static import